docs/index.html

```diff
@@ -57,81 +57,146 @@ <h1>To use this website, JavaScript needs to be enabled.</h1>
     <header class="clearfix">
       <h1>Devicon <span><a href="https://github.com/devicons/devicon/releases">{{ latestReleaseTagging }}</a></span></h1>
       <h2>Devicon is a set of icons representing programming languages, designing & development tools. You can use it as a font or directly copy/paste the svg code into your project.</h2>
+      <i>*Please read the legal conditions of using the logos here at the bottom of this header</i>
```

Comment:
Is this `*` added intentionally or is it a typo?

Reply:
It's intentional. For me, it works as an attention grabber but we can remove it if you want.